### PR TITLE
Link geo entities to geo_data

### PR DESCRIPTION
**Pull request by renanbr**

### CODE REVIEW

File: src/Entity/Geo/GeoTrait.php [hunk 1]
```diff
@@ -36,6 +37,14 @@ trait GeoTrait
      */
     private $name;
 
+    /**
+     * @var GeoData|null
+     *
+     * @ORM\OneToOne(targetEntity="App\Entity\GeoData")
+     * @ORM\JoinColumn(nullable=true)
```
Comment (ottaviano) [inline src/Entity/Geo/GeoTrait.php:44]: we can avoid this annotation as this is nullable by default

File: src/Repository/Geo/ZoneRepository.php [hunk 2]
```diff
@@ -50,6 +50,7 @@ public function zoneableAsZone(ZoneableInterface $zoneable): Zone
 
         $zone->activate($zoneable->isActive());
         $zone->setName($zoneable->getName());
+        $zone->setGeoData($zoneable->getGeoData());
```
Comment (ottaviano) [inline src/Repository/Geo/ZoneRepository.php:53]: It seems that `ZoneableInterface` has not `getGeoData`, same for others methods getName, getCode, isActive 🤔

Comment (renanbr) [reply src/Repository/Geo/ZoneRepository.php:53]: `ZoneableInterface` extends `GeoInterface`, thus they are part of the contract

Comment (ottaviano) [reply src/Repository/Geo/ZoneRepository.php:32]: oh yes, my bad, sorry

**Review by ottaviano — APPROVED** [review]

**Review by ottaviano — APPROVED** [review]

**Pull request merged**